application/src/main/kotlin/no/kartverket/matrikkel/bygning/application/models/Bygning.kt

```diff
@@ -24,7 +25,8 @@ data class Multikilde<T : Any>(val autoritativ: T? = null, val egenregistrert: T
 
 data class RegisterMetadata(
     val registreringstidspunkt: Instant,
-    val registrertAv: RegistreringAktoer
+    val registrertAv: RegistreringAktoer,
+    val kildematriale: KildematerialeKode? = null
```

Comment:
```suggestion
    val kildemateriale: KildematerialeKode? = null
```

Litt blanding akkurat nå med kildemateriale vs kilde. Bra å få det konsekvent (foreslår å gå for kildemateriale)